src/profile/v100/address/non_fungible_local_id_uniffi_fn.rs

```diff
@@ -63,6 +63,11 @@ pub fn new_non_fungible_local_id_sample_other() -> NonFungibleLocalId {
     NonFungibleLocalId::sample_other()
 }
 
+#[uniffi::export]
+pub fn new_non_fungible_local_id_random() -> NonFungibleLocalId {
+    NonFungibleLocalId::random()
```

Comment:
@micbakos-rdx and when we merge your PR https://github.com/radixdlt/sargon/pull/50 we can create a `random()` func for `NonFungbileGlobalId` too!